<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NESettingWidget</class>
 <widget class="QWidget" name="NESettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>397</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>408</width>
    <height>397</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>422</width>
    <height>397</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#NESettingWidget{
	background-color: rgb(255, 255, 255);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>24</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font-size: 16px</string>
          </property>
          <property name="text">
           <string>视频设置</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>20</number>
          </property>
          <property name="verticalSpacing">
           <number>20</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font-size: 14px</string>
            </property>
            <property name="text">
             <string>分辨率</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboxProfile">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
    border:1px solid #d9d9db;
    border-radius: 2px;
    padding-left: 12px;
	font-size: 14px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 8px;
	border-left-style: solid;
    border-left-width: 1px;
	padding-right: 12px;
}

QComboBox::down-arrow {
	image: url(:/image/triangle-combox.png);
    max-width: 8px;
    max-height: 4px;
}


</string>
            </property>
            <item>
             <property name="text">
              <string>160*90</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>320*180</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>640*360</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1280*720</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1920*1080</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">font-size: 14px</string>
            </property>
            <property name="text">
             <string>帧率</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboxFramerate">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
    border:1px solid #d9d9db;
    border-radius: 2px;
    padding-left: 12px;
	font-size: 14px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 8px;
	border-left-style: solid;
    border-left-width: 1px;
	padding-right: 12px;
}


QComboBox::down-arrow {
	image: url(:/image/triangle-combox.png);
    max-width: 8px;
    max-height: 4px;
}

</string>
            </property>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(230, 231, 235);</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font-size: 16px</string>
          </property>
          <property name="text">
           <string>音频设置</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="horizontalSpacing">
           <number>32</number>
          </property>
          <property name="verticalSpacing">
           <number>20</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">font-size: 14px</string>
            </property>
            <property name="text">
             <string>场景</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>24</number>
            </property>
            <item>
             <widget class="QRadioButton" name="rBMusic">
              <property name="styleSheet">
               <string notr="true">QRadioButton {
	min-width: 14px;
	min-height: 14px;
    font-size: 14px;
}

QRadioButton::indicator {
	width: 14px;
	height: 14px;
}

QRadioButton::indicator::unchecked {
	border-image: url(:/image/radio-uncheck.png);
}

QRadioButton::indicator::checked {
	
	border-image: url(:/image/radio-check.png);
}</string>
              </property>
              <property name="text">
               <string>音乐</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rBVoice">
              <property name="styleSheet">
               <string notr="true">QRadioButton {
	min-width: 14px;
	min-height: 14px;
    font-size: 14px;
}

QRadioButton::indicator {
	width: 14px;
	height: 14px;
}

QRadioButton::indicator::unchecked {
	border-image: url(:/image/radio-uncheck.png);
}

QRadioButton::indicator::checked {
	
	border-image: url(:/image/radio-check.png);
}</string>
              </property>
              <property name="text">
               <string>语音</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">font-size: 14px</string>
            </property>
            <property name="text">
             <string>音质</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboxAudioQuality">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
    border:1px solid #d9d9db;
    border-radius: 2px;
    padding-left: 12px;
	font-size: 14px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 8px;
	border-left-style: solid;
    border-left-width: 1px;
	padding-right: 12px;
}


QComboBox::down-arrow {
	image: url(:/image/triangle-combox.png);
    max-width: 8px;
    max-height: 4px;
}

</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 231, 235);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>52</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>239</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>32</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 14px;
border-radius: 2px;
background-color: #FFFFFF;
border: 1px solid #D9D9DB;</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConfirm">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 14px;
border-radius: 2px;
color: #FFFFFF;
background-color: rgb(51, 126, 255);</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
